src/boss/config.go

```diff
@@ -47,6 +49,7 @@ func LoadConf(path string) error {
 	}
 
 	cloudvm.LoadGcpConfig(Conf.Gcp)
```

Comment:
I'm not sure I understand why we need this.  When we unmarshall the config, won't we already get the the sub configs from the file?